packages/database_v2/__WIP_pg_schema_review/artifacts/provision.sql

```diff
@@ -0,0 +1,222 @@
+-- Generated by prisma gainst the schema.prisma file
+
+-- CreateEnum
+CREATE TYPE "resource_status" AS ENUM ('Idle', 'PinQueued', 'Pinned', 'FailedURIParse', 'FailedFetch', 'PinFailure');
+
+-- CreateEnum
+CREATE TYPE "token_asset_status" AS ENUM ('Queued', 'Failed', 'Succeeded');
+
+-- CreateEnum
+CREATE TYPE "pin_status" AS ENUM ('PinFailed', 'Pinned', 'Pinning', 'PinQueued');
+
+-- CreateTable
+CREATE TABLE "block" (
+    "hash" TEXT NOT NULL,
+    "number" INTEGER NOT NULL,
+
+    PRIMARY KEY ("hash")
+);
+
+-- CreateTable
+CREATE TABLE "token" (
+    "id" TEXT NOT NULL,
+    "token_id" TEXT NOT NULL,
+    "mint_time" TIMESTAMP(3) NOT NULL,
+    "token_asset_id" TEXT NOT NULL,
+    "token_contract_id" TEXT NOT NULL,
+    "owner_id" TEXT NOT NULL,
+
+    PRIMARY KEY ("id")
+);
+
+-- CreateTable
+CREATE TABLE "tokens_on_blocks" (
+    "block_hash" TEXT NOT NULL,
+    "token_id" TEXT NOT NULL,
+    "created" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+
+    PRIMARY KEY ("block_hash","token_id")
+);
+
+-- CreateTable
+CREATE TABLE "token_asset" (
+    "id" TEXT NOT NULL,
+    "token_uri" TEXT NOT NULL,
+    "ipnft" TEXT NOT NULL,
+    "problem" TEXT,
+
+    PRIMARY KEY ("id")
+);
+
+-- CreateTable
+CREATE TABLE "metadata" (
+    "id" TEXT NOT NULL,
+    "cid" TEXT,
+    "source_id" TEXT NOT NULL,
+    "name" TEXT,
+    "description" TEXT,
+    "imageId" TEXT NOT NULL,
+    "image_uri" TEXT NOT NULL,
+
+    PRIMARY KEY ("id")
+);
+
+-- CreateTable
+CREATE TABLE "resource" (
+    "status" "resource_status" NOT NULL,
+    "status_text" TEXT NOT NULL,
+    "uri" TEXT NOT NULL,
+    "ipfs_url" TEXT NOT NULL,
+    "problem" TEXT NOT NULL,
```

Comment:
This seems like leftover from older schema it no longer exists

https://github.com/ipfs-shipyard/nft.storage/blob/343b835dc902454b91e702e2845e9f0ec6cae471/packages/database/fauna/resources/schema.graphql#L250-L308

Reply:
ok yes, it looks like status and status_text are the new entities, and I've updated the enum as well.